src/App.tsx

```diff
@@ -1,61 +1,115 @@
+import { ChangeEvent, FormEventHandler, useState } from 'react';
 import './App.scss';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todo } from './types/Todo';
+import { User } from './types/User';
+
+function getUserById(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+let highestId = Math.max.apply(null, todosFromServer.map(todo => todo.id));
```

Comment:
Avoid using `Math.max.apply(null, ...)` to find the highest id. Use `Math.max(...todosFromServer.map(todo => todo.id))` instead